sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,54 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-02-11)
```

Comment:
There was also a renaming of the batch methods were we moved `batch` as the suffix